mission1/index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>판다마켓</title>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE-edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href="img/panda.png">
+    <link rel="stylesheet" href="style.css">
+    <meta property="og:title" content="panda-market">
+    <meta property="og:url" content="https://panda-market-second-hands.netlify.app/">
+    <meta property="og:type" content="website">
+    <meta property="og:image" content="img/Property 1=lg@3x.png">
+    <meta property="og:description" content="1등 중고거래는 역시 판다!">
+    <script>
+      (function(d) {
+        var config = {
+          kitId: 'hud6cvy',
+          scriptTimeout: 3000,
+          async: true
+        },
+        h=d.documentElement,t=setTimeout(function(){h.className=h.className.replace(/\bwf-loading\b/g,"")+" wf-inactive";},config.scriptTimeout),tk=d.createElement("script"),f=false,s=d.getElementsByTagName("script")[0],a;h.className+=" wf-loading";tk.src='https://use.typekit.net/'+config.kitId+'.js';tk.async=true;tk.onload=tk.onreadystatechange=function(){a=this.readyState;if(f||a&&a!="complete"&&a!="loaded")return;f=true;clearTimeout(t);try{Typekit.load(config)}catch(e){}};s.parentNode.insertBefore(tk,s)
+      })(document);
+    </script>
+  </head>
+  <body>
+    <!-- header -->
+    <header>
+      <div class="header-logo">
+        <div class="logo-login">
```

Comment:
```suggestion
      <div class="header-logo">
        <div class="logo-login">
```

이 둘을 분리한 이유가 무엇인가요? / 굳이 생성한 이유는 무엇인가요? 

이 마크업을 생각해보았을때, 디자인적 요구사항을 고려해보면 
1. 로고와 로그인버튼이 최대한 펼쳐질 수 있는 최대 넓이가 존재한다
2. 로고는 펼쳐진 넓이에서 제일 좌측에 위치하고, 로그인버튼은 우측에 존재한다

로 보입니다. 이 관점에서 header-logo / logo-login으로 분리할 필요는 없어보이고, 오히려 다음과 같이해보면 어떨런지요

```html
<header class="main-header">
  <a class="main-header__logo">
    <img class="main-header__logo-img" />
    판다마켓
  </a>
  <a class="main-header__login">로그인</a>
</header>
```

그리고 스타일은 다음과 같이 잡는거죠

```css
/**
* 메인 헤더 스타일
*/

.main-header {
  width: 100%;
  max-width: 1100px;
  margin: 0 auto;  // 최대 넓이로 펼쳐주고, 상위 콘텐츠 기준 중앙정렬
  display: flex;
  justify-content: space-between; // 내부 요소들을 최대한 서로 멀리 떨어뜨려놓는다
}
...
```